shared/login/recover-password/prompt-reset/container.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import * as Container from '../../../util/container'
+import * as RecoverPasswordGen from '../../../actions/recover-password-gen'
+import DeviceSelector from '.'
+import {InfoIcon} from '../../../signup/common'
+
+type OwnProps = {}
+
+const ConnectedPromptReset = Container.connect(
```

Comment:
Curious why you went with `connect` rather than hooks for these?

Reply:
Other than this being old code, I find it somewhat hard to figure out where to use hooks for redux state. Would it mean that I wouldn't have a container.tsx? How would the stories work then?

Reply:
Yeah, you wouldn't have `container.tsx` for these then. Stories will still work but i believe you need to add a mock store. Device stories are a good example.
https://github.com/keybase/client/blob/e3d61efb87746b074c1f0b834236750597451f6b/shared/devices/device-page/index.stories.tsx#L61